components/mixins/opportunity/opportunity-person-details.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/opportunity-person-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "opportunity Person Relation Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/classes/opportunity-person"
+  ],
+  "description": "XDM business opportunity person relation details.",
```

Comment:
Please update the description so it describes what use-case should use this mixin.
If you have no specific use-case, the move these fields to the class.